src/components/organisms/BalanceGameSection/BalanceGameSection.tsx

```
@@ -1,242 +1,240 @@
+import Chips from '@/components/atoms/Chips/Chips';
+import Divider from '@/components/atoms/Divider/Divider';
+import { SubTag } from '@/components/atoms/SubTag/SubTag';
+import MenuTap, { MenuItem } from '@/components/atoms/MenuTap/MenuTap';
+import GameStageBar from '@/components/atoms/GameStageBar/GameStageBar';
+import InteractionButton from '@/components/atoms/InteractionButton/InteractionButton';
+import ToastModal from '@/components/atoms/ToastModal/ToastModal';
+import ShareModal from '@/components/molecules/ShareModal/ShareModal';
+import LoginModal from '@/components/molecules/LoginModal/LoginModal';
+import BalanceGameBox from '@/components/molecules/BalanceGameBox/BalanceGameBox';
+import useToastModal from '@/hooks/modal/useToastModal';
+import { useBalanceGameBookmark } from '@/hooks/mypages/useBalanceGameBookmark';
```

ESLint

[error] `@/hooks/mypages/useBalanceGameBookmark` import should occur after import of `@/types/vote`

(import/order)